cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -98,6 +132,17 @@ data KeyCmd
   | KeyConvertITNBip32ToStakeKey SomeKeyFile OutputFile
   deriving (Eq, Show)
 
+renderKeyCmd :: KeyCmd -> Text
+renderKeyCmd cmd =
+  case cmd of
+    KeyGetVerificationKey _ _ -> "key verification-key"
+    KeyNonExtendedKey  _ _ -> "key non-extended-key"
+    KeyConvertByronKey _ _ _ _ -> "key convert-byron-key"
+    KeyConvertByronGenesisVKey _ _ -> "key convert-byron-genesis-key"
+    KeyConvertITNStakeKey _ _ -> "key convert-itn-key"
+    KeyConvertITNExtendedToStakeKey _ _ -> "key convert-itn-extended-key"
```

Comment:
Words on LHS and RHS don't match?

Reply:
Sorry I don't understand what you mean? I've mapped the `KeyCmd` type to what is display in the cli executable.